### PR TITLE
Metric configuration retrieval

### PR DESCRIPTION
**Pull request by rafamanzo**

### CODE REVIEW

File: features/kalibro_configuration/hotspot_metric_configurations.feature [hunk 1]
```diff
@@ -0,0 +1,12 @@
+Feature: HotspotMetricConfiguration retrieval
+  In order to be able to list just with the HotspotMetricConfigurations
+  As a developer
+  I want to get all HotspotMetricConfigurations of a given KalibroConfiguration
+
+  @kalibro_configuration_restart @skip
```
Comment (diegoamc) [inline features/kalibro_configuration/hotspot_metric_configurations.feature]: Can we remove `@skip`?

File: features/kalibro_configuration/hotspot_metric_configurations.feature [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  Scenario: get a list of all metric configurations of some kalibro configuration
+    Given I have a kalibro configuration with name "Kalibro for Java"
+    And I have a reading group with name "Group"
```
Comment (diegoamc) [inline features/kalibro_configuration/hotspot_metric_configurations.feature]: Hotspot metric configurations do not need reading groups.

**Comment by rafamanzo** [comment]
All is fixed. Thanks for the review :+1:

**Comment by dread-uo** [comment]
:+1:

**Pull request merged**